python_scripts/api_requests.py

```diff
@@ -117,10 +168,10 @@ def daterange(start_date, end_date):
     return daterange(start_date, end_date)
 
 
-def get_bookings_df(bookings):
-    bookings_df = pd.io.json.json_normalize(bookings)
-    renames = read_json("metadata/bookings_renames.json")
-    bookings_metadata = read_json("metadata/bookings.json")
+def get_bookings_df(bookings, db_version, env):
+    bookings_df = pd.json_normalize(bookings)
+    renames = read_json(f"metadata/{db_version}/{env}/bookings_renames.json")
+    bookings_metadata = read_json(f"metadata/{db_version}/{env}/bookings.json")
```

Comment:
To handle db version and environment.
Also removed `io.json` as this was depreceated in later version of pandas